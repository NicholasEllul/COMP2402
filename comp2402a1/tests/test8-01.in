Here is some
test code
test code
lol hello
this is test code
test code
Here is some
Aloha mundo
Jello World
Jello

Bob

Bob

Yoda
OKAYYY
