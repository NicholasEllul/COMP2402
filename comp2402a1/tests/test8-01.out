
test code
Bob
Here is some
Aloha mundo
Jello
Jello World
OKAYYY
Yoda
lol hello
this is test code
